ParisBNF/abb/BNFabb190.xml

```diff
@@ -0,0 +1,411 @@
+                       <extent>
+                          <measure unit="leaf" quantity="16">16</measure>
+                          <dimensions type="outer" unit="mm">
+                             <height>194</height>
+                             <width>132</width>                      
+                          </dimensions>                       </extent>
+
+                   
+                         <foliation>Foliation of the Library in the upper right corner of the folia.</foliation>
+                       <collation>
+                          <note>The volume is made out of 
+                             dispersed leaves, originating from various manuscripts, mounted on paper guards and placed in an European binding. 
+                             Something might be missing in the manuscript, 
+                          since A. d'Abbadie refers in his catalogue to two more texts (numbers 2 and 4, all in all 16 folia) which are not 
+                          in the volume in its present shape and were not there by the time of the creation of other two catalogues.</note>
+                       </collation>
+                       <condition key="good"/>
+                    </supportDesc>
+                 </objectDesc>
+                 <bindingDesc>
+                    <binding contemporary="false" xml:id="binding">
+                       <decoNote xml:id="b1">European binding.
+                       </decoNote>                        
+                       <decoNote xml:id="b2" type="bindingMaterial">
+                          <material key="cardboard"/>
+                            <material key="paper"/>
+                            </decoNote>
+                          </binding>
+                 </bindingDesc>
+              </physDesc>
+
+              <history>
+                 <origin>
+                   <origDate notBefore="1750" notAfter="1848"/>
+                   All units were produced approximately in the same period, 
+                   late 18th/first half of the 19th-century.
+                   </origin>
+                   <provenance>The leaves were aquired by
+                     <persName ref="PRS1071dAbbadi" role="owner">Antoine d'Abbadie</persName>
+                     sometime between <date>1840</date> and <date>1848</date>.                                       
+                   </provenance>
+                <acquisition>The leaves entered <placeName ref="INS0303BNF"/> in
+                     <date>1902</date>. 
+                     </acquisition>
+                 </history>
+
+              <msPart xml:id="p1">
+                 <msIdentifier/>
+                 <msContents>
+                    <summary/>
+
+                    <msItem xml:id="p1_i1">
+                       <locus from="1ra" to="2vb"/>
+                      <title type="incomplete" cert="medium">A story about the finding of The Holy Cross featuring Empress Helena and Cyriac (the beginning is mising)</title>
```

Comment:
missing

I would probably put the beginning is missing as a note, or write (acephalous)